lib/matplotlib/collections.py

```diff
@@ -2019,6 +2019,15 @@ def set_paths(self):
     def get_datalim(self, transData):
         return (self.get_transform() - transData).transform_bbox(self._bbox)
 
+    def get_coordinates(self):
+        """
+        Return the x and y vertices of the mesh.
+
+        The shape of the returned coordinates is
+        (*meshWidth* + 1, *meshHeight* + 1, 2).
```

Comment:
Does the user know what *meshWidth/Height* are?  

Maybe a bit more explanation?
```suggestion
        (*meshWidth* + 1, *meshHeight* + 1, 2), where *meshWidth* and *meshHeight* are the size of the 
        mesh, passed in when the Artist was initialized. 
```

Reply:
Hmm... Good point. Unfortunately, the initialized wording isn't entirely clear to me either because `pcolormesh` can auto-expand the coordinates for you (which is why I wanted this public in the first place). The docstring for QuadMesh gives some info on what meshWidth/meshHeight are.
https://matplotlib.org/stable/api/collections_api.html#matplotlib.collections.QuadMesh